velox/expression/ExprToSubfieldFilter.h

```diff
@@ -398,7 +398,7 @@ between(const Timestamp& min, const Timestamp& max, bool nullAllowed = false) {
 }
 
 inline std::unique_ptr<common::TimestampRange> lessThan(
-    Timestamp& max,
+    Timestamp&& max,
```

Comment:
`const Timestamp&`

Reply:
The value needs to be modified inside the function.

Reply:
Actually let's just copy and modify the copy here.